x-pack/plugin/ccr/src/main/java/org/elasticsearch/xpack/ccr/action/ShardFollowNodeTask.java

```diff
@@ -47,7 +54,7 @@
 public abstract class ShardFollowNodeTask extends AllocatedPersistentTask {
 
     private static final int DELAY_MILLIS = 50;
-    private static final Logger LOGGER = Loggers.getLogger(ShardFollowNodeTask.class);
+    private static final Logger LOGGER = LogManager.getLogger(ShardFollowNodeTask.class);
```

Comment:
I am not sure why you changed this.

Reply:
It is an unrelated change. `Loggers#getLogger(...)` is deprecated and I just thought that I would change it.

Reply:
ok, thanks.